code/style.css

```diff
@@ -2,6 +2,298 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
+
+html {
+  Font-Family: 'Playfair Display', Serif;
+  }
+  
+body  {
+  background-color: #FAF0E6;
+  margin: 0;
+}
+    
+.hero-img {
+  background-image: url(./img/coffe_extreme.jpg);
+  height:350px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover; 
+}
+    
+  
+.hero-img h1 {
+  position: relative;
+  text-align: center;
+  font-size: 35px;
+  color: white;
+  margin:0;
+}
+  
+  /*WE ARE TEXT */
+  
+.weare {
+  display: flex;
+  text-align: center;
+  margin-top: 5px;
+  color: black;
+  font-style: italic;
+  font-size: 10px;
+  text-decoration: underline solid white; 
+}
+  
+  /*sign up text */
+  
+.sign-up {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  margin-top:30px;
+}
+  
+    /*Container main content*/
+  
+.container{
+  display: inline-flex;
+  flex-direction: column;
+  margin-left:5px;
+  margin-bottom: 30px;
+  margin-top: 30px;
+}
+  
+  /*Form */
+  
+form {
+  display: flex;
+  flex-direction: column;
+  height: 600px;
+  padding-top: 30px;
+  border-radius: 8px;
+  box-shadow: 2px 2px 2px 2px rgba(0,0,0,0.15);
+  font-size: 15px;
+}
+    
+.form-group {
+  display: flex;
+  justify-content: center;   
+}
+  
+.form-group label {
+  justify-content: center;
+  padding:10px;
+  border-radius:10px;
+}
+  
+.form-group-button {
+  display: flex;
+  justify-content: center;
+}
+  
+h3 {
+  display: flex;
+  justify-content: center;
+}
+  
+button {
+  height: 50px;
+  background-color: white;
+  border-radius: 15px;
+  border-style: hidden;
+  font-size: 20px;
+  Font-Family: 'Playfair Display', Serif;
+  box-shadow: grey 2px 2px;
+}
+  
+input {
+  font-size: 20px;
+}
+  
+  /*Color Validations */ 
+  
+input:valid {
+  background-color: #63C380;
+}
+  
+input:invalid {
+  border-color:#FE6C6C ;
+}
+  
+label {
+  display: block;
+  margin-bottom: 30px;
+}
+  
+  
+   /* Accordion */
+  
+.answer {
+  display: none;
+}
+  
+   /* Accordion  box styling and placement*/
+  
+.accordion {
+  display: flex;
+  flex-direction: column;
+  height: 600px;
+  margin-left: 5px;
+  margin-top: 100px;
+  margin-bottom:90px ;
+  border-radius: 8px;
+  box-shadow: 2px 2px 2px 2px rgba(0,0,0,0.15);
+}
+  
+.accordion h1 {
+  display: flex;
+  text-align: center;
+  font-size: 30px;
+  background-color:rgb(250, 240, 230);;
+}
+  
+.answer {
+  height: 0px;
+  opacity: 0;
+  overflow: hidden;
+  transition: all 1s ease-in-out;
+  -webkit-transition: all 1s ease-in-out;
+  -moz-transition: all 1s ease-in-out;
+  -ms-transition: all 1s ease-in-out;
+  -o-transition: all 1s ease-in-out;
+}
+  
 .active + .answer {
-  /* some styling here */
+  display: inline-block;
+  opacity: 1;
+  height: 80px;
+  background-color: #FAEBD7;
+}
```

Comment:
To create a smoother transition you can use the Inspector tool to check what's changing when you click the FAQ and then add the transition property. A bit of reading here but this also looks good:
http://css3.bradshawenterprises.com/accordions/